src/style.css

```diff
@@ -1 +1,95 @@
-/* add styles here */
+* {
+  outline: none;
+  box-sizing: border-box;
+}
+
+:root {
+  --text-color: #3d4e61;
+  --color: #687c94;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+form {
+  margin: 20px 0;
+}
+
+.big--input {
+  padding-left: 62px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
+  color: var(--text-color);
```

Comment:
```suggestion
  font-size: 16px;
  font-weight: 300;
  color: var(--text-color);
```
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/96265844/130e0a7a-37b1-40e6-a1e1-bab70ad108
here i delete it and all look ok